charts/dev/livebook/questions.yaml

```diff
@@ -9,31 +9,56 @@ questions:
-                                        schema:
-                                          type: string
-                                          required: true
-                                          private: true
-                                      - variable: LIVEBOOK_DEBUG
-                                        label: "Debug Logging (LIVEBOOK_DEBUG)"
-                                        description: >
-                                          enables verbose logging, when set to "true". Disabled by default.
-                                        schema:
-                                          type: boolean
-                                      - variable: LIVEBOOK_UPDATE_INSTRUCTIONS_URL
-                                        label: "Update instruction URL (LIVEBOOK_UPDATE_INSTRUCTIONS_URL)"
-                                        description: >
-                                          sets the URL to direct the user to for updating Livebook when a new version becomes available.
-                                        schema:
-                                          type: string
+# Include{containerBasic}
+# Include{containerAdvanced}
+# Include{containerConfig}
+# Include{podOptions}
+  - variable: livebookPassword
+    group: "App Configuration"
+    label: "Password (LIVEBOOK_PASSWORD)"
+    description: Password needed to access livebook (must be at least 12 characters)
+    schema:
+      type: string
+      min_length: 12
+      required: true
+      private: true
+  - variable: livebookDebug
+    group: "App Configuration"
+    label: "Debug Logging (LIVEBOOK_DEBUG)"
+    description: >
+      Enables verbose logging, when set to "true". Disabled by default.
+    schema:
+      type: boolean
+  - variable: livebookUpdateInstructionsUrl
+    group: "App Configuration"
+    label: "Update instruction URL (LIVEBOOK_UPDATE_INSTRUCTIONS_URL)"
+    description: >
+      Sets the URL to direct the user to for updating Livebook when a new version becomes available.
+    schema:
+      type: string
+  - variable: livebookAwsCredentials
+    group: "App Configuration"
+    label: "Read AWS Credentials From Env (LIVEBOOK_AWS_CREDENTIALS)"
+    description: >
+      Enable Livebook to read AWS Credentials from environment variables,
+      AWS Credentials, EC2/ECS metadata when configuring S3 buckets.
+    schema:
+      type: boolean
+  - variable: awsAccessKeyId
+    group: "App Configuration"
+    label: "AWS Access Key ID"
+    description: >
+      AWS Access Key ID in case you want to configure AWS S3 Storage inside Livebook
+    schema:
+      type: string
+  - variable: awsSecretAccessKey
+    group: "App Configuration"
+    label: "AWS Secret Access Key"
+    description: >
+      AWS Secret Access Key in case you want to configure AWS S3 Storage inside Livebook
+    schema:
+      type: string
+      private: true
```

Comment:
I'd prefer something like this instead
livebook:
  password:
  awsCredentials:
  ...
  
Keeps it organized

Reply:
You mean password instead of livebookPassword? Removing the prefix? Sure can do. But what do you mean "livebook:"?

Reply:
forgot to put it inside fences :D

Edited, now should be more clear !

Reply:
Ah I see. Or just ommit livebook. This is the livebool chart after all...

Reply:
Please put values in the object. Having values in the values.yaml file can potentially conflict with dependency charts.

Reply:
Better now? Is this what you meant?